### DISCUSSION TITLE
pytest parameterized test executions are shown as retries in allure test report

### DISCUSSION THREAD

**Discussion by ndomnin** [allure-framework/allure2#1738] [TestOps]
Hello,

I've tried to set `@allure.id()` to existing parametrized test to get more organized test executions structure, but got all execution shown as retries:
![image](https://user-images.githubusercontent.com/3264448/184837177-3b6ca259-252d-422d-8926-3830bf3161c2.png)

Coud you please advise something with examples of how it could be done in this case:
```
@allure.id("123")
@pytest.mark.scenario("allure/sample/test_allure_sample")
@pytest.mark.parametrize("config",  basic_configs(), indirect=True)
def test_allure_sample(config: Config):
    """
    Sample test description
    """
    step1()
    step2()
    step3()
```

**Comment by cheshi-mantu** [comment]
@ndomnin hi there.
this question is related to Allure TestOps, which is a commercial product supported by via https://help.qameta.io by a dedicated team.
We'll be happy to assist.
Please consider registering with your corporate email (domain is to be of a company owning either commercial or trial licence).
we are going to need the following:
- Allure TestOps release ID (exact one) which can be retrieved as described [here](https://help.qameta.io/support/solutions/articles/101000391823-how-do-i-get-the-release-id-of-allure-testops-instance-)
- releases ids if pytest, allure-pytest
- working example with 1 dummy or real test as link to GH public repo.
thanks.
